src/_main_/settings.py

```diff
@@ -286,3 +290,16 @@
 STATICFILES_LOCATION = 'static'
 MEDIAFILES_LOCATION = 'media'
 
+
+# Celery Setup
+CELERY_BROKER_URL = os.environ.get("CELERY_BROKER_URL")
```

Comment:
How do we create one of these?

Reply:
First of all, these packages have to be installed:
- redis
- celery
- django-celery-beat
- django-celery-results

After installing run `brew services start redis ` to start the Redis server (celery broker).
Add the redis server url to your env file like so
`CELERY_BROKER_URL = redis url` 
eg. ` CELERY_BROKER_URL='redis://localhost:6379/0'`